static/src/javascripts/projects/common/modules/experiments/tests/giraffe.js

```diff
@@ -33,7 +33,8 @@ define([
         this.dataLinkNames = '';
         this.idealOutcome = '';
         this.canRun = function () {
-             return window.guardian.config.page.edition == 'UK';
+             return !window.guardian.config.page.isSensitive &&
```

Comment:
So, in `commercial` we already have [commercial-features.js](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/commercial-features.js), which has `sensitiveContent` and other definitions defined already. Feel free to use!